include/private/tdymeshimpl.h

```diff
@@ -133,6 +133,7 @@ struct _TDy_mesh {
 
 };
 
+PETSC_EXTERN PetscErrorCode OutputMesh(TDy);
```

Comment:
Should these calls (and 137-8) be prefixed with TDy to avoid possible name collisions?

Reply:
This is a good catch. But, I now notice that there are many places in the lib that I have not added a prefix `TDy`. How about if I fix the lib by adding `TDy` prefix in a separate PR?